i18n/es.json

```diff
@@ -295,18 +411,46 @@
     "id": "api.channel.rename_channel.cant_rename_group_messages.app_error",
     "translation": "No puedes renombrar un canal de grupo"
   },
+  {
+    "id": "api.channel.restore_channel.restored.app_error",
+    "translation": "No se puede archivar el canal. El canal no ha sido archivado."
```

Comment:
I can't edit this in translate.mattermost.com because is now down (I don't know why, but this string should be different)
Something like `No se puede restaurar el canal. El canal no ha sido archivado.`